server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPConfiguration.java

```diff
@@ -164,4 +172,8 @@ public Version getDefaultVersion() {
     public Optional<Long> getMaximumSendSize() {
         return maximumSendSize;
     }
+
+    public boolean supportsDelaySends() {
```

Comment:
Where do we use it?
`EmailSubmissionSetMethod` or somewhere?

Reply:
Well, I think this method is unnecessary

Reply:
I already see `supportsDelaySends` config property in `JmapRfc8621Configuration`, should not add the same property in `JMAPConfiguration`.